xtask/src/cbindgen.rs

```diff
@@ -308,7 +308,7 @@ fn gen_corelib(root_dir: &Path, include_dir: &Path) -> anyhow::Result<()> {
             r"
 namespace sixtyfps {{
     namespace private_api {{ enum class VersionCheck {{ Major = {}, Minor = {}, Patch = {} }}; class ComponentWindow; }}
-    namespace cbindgen_private {{ using sixtyfps::private_api::ComponentWindow; using namespace vtable; struct KeyEvent; }}
+    namespace cbindgen_private {{ using sixtyfps::private_api::ComponentWindow; using namespace vtable; struct KeyEvent; using namespace private_api; }}
```

Comment:
what's the reason for this change?

Reply:
For the structs for different items, cbindgen maps `width: Property<f32>` to `Property<f32> width`. Since the generated structs live in `cbindgen_private` this was an easy way for that to compile. If you prefer I can also look for an alternate way so that cbindgen generates `private_api::Property<f32>` instead.

Reply:
I guess it's fine like that.

Maybe one want to be a bit more explicit and only do `using private_api::Property;`

Reply:
Done